### ISSUE TITLE
Provisioning profile "iOS Team Provisioning Profile: xxx" doesn't include signing certificate "Apple Development: XXX"

### ISSUE THREAD

**Issue by KananAbilzada** [fastlane/fastlane#19553]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

I have created certificates and I can upload the app to TestFlight manually, that I tested it. But when using this:
gym(
      project: "xxx.xcodeproj",
      scheme: "xxx",
      clean: true,
      export_xcargs: "-allowProvisioningUpdates",
      output_name: "ReleaseApp.ipa"
    )
    

##### Command executed
gym(
      project: "xxx.xcodeproj",
      scheme: "xxx",
      clean: true,
      export_xcargs: "-allowProvisioningUpdates",
      output_name: "ReleaseApp.ipa"
    )

##### Complete output when running fastlane, including the stack trace and command used
user@user-iMac xxx % fastlane _upload

[✔] 🚀 
[15:06:54]: fastlane detected a Gemfile in the current directory
[15:06:54]: However, it seems like you didn't use `bundle exec`
[15:06:54]: To launch fastlane faster, please use
[15:06:54]: 
[15:06:54]: $ bundle exec fastlane _upload
[15:06:54]: 
[15:06:54]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
[15:06:56]: ------------------------------
[15:06:56]: --- Step: default_platform ---
[15:06:56]: ------------------------------
[15:06:56]: Driving the lane 'ios _upload' 🚀
[15:06:56]: -----------------
[15:06:56]: --- Step: gym ---
[15:06:56]: -----------------
[15:06:56]: Resolving Swift Package Manager dependencies...
[15:06:56]: $ xcodebuild -resolvePackageDependencies -scheme xxx -project xxx.xcodeproj
[15:06:58]: ▸ Command line invocation:
[15:06:58]: ▸     /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -resolvePackageDependencies -scheme xxx -project xxx.xcodeproj
[15:06:58]: ▸ User defaults from command line:
[15:06:58]: ▸     IDEPackageSupportUseBuiltinSCM = YES
[15:06:58]: ▸ --- xcodebuild: WARNING: Using the first of multiple matching destinations:
[15:06:58]: ▸ { platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Any iOS Device }
[15:06:58]: ▸ { platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Any iOS Simulator Device }
[15:06:58]: ▸ { platform:iOS Simulator, id:73ABE1B8-2851-4508-BC24-4EE139E3E529, OS:11.4, name:iPhone 5s }
[15:06:58]: ▸ { platform:iOS Simulator, id:ADDA00E3-C02F-45AE-BAC7-804D4DED94A3, OS:11.4, name:iPhone 6 }
[15:06:58]: ▸ { platform:iOS Simulator, id:7E4B4D2A-2291-44E2-B6E1-5BC744074961, OS:11.4, name:iPhone 6 Plus }
[15:06:58]: ▸ { platform:iOS Simulator, id:98A75BE0-1E9F-4BFC-8553-5517FFA2D3A3, OS:11.4, name:iPhone 6s }
[15:06:58]: ▸ { platform:iOS Simulator, id:4C5B3EE7-F6B5-44E7-A7E1-960B470FB72D, OS:11.4, name:iPhone 6s Plus }
[15:06:58]: ▸ { platform:iOS Simulator, id:61763B25-035F-4008-B51A-1CE1D9EF54D0, OS:11.4, name:iPhone 7 }
[15:06:58]: ▸ { platform:iOS Simulator, id:CA248D3F-0B38-4856-AE29-E01127705DF3, OS:11.4, name:iPhone 7 Plus }
[15:06:58]: ▸ { platform:iOS Simulator, id:5855F737-BD6A-4ECF-9609-3AB8EC2E1EA1, OS:11.4, name:iPhone 8 }
[15:06:58]: ▸ { platform:iOS Simulator, id:368AEC8B-1298-44E7-A16A-8AA6D43AE138, OS:14.5, name:iPhone 8 }
[15:06:58]: ▸ { platform:iOS Simulator, id:C58C1F33-63D1-490E-AE7C-5FD63DB2A40F, OS:15.0, name:iPhone 8 }
[15:06:58]: ▸ { platform:iOS Simulator, id:4B578975-16C5-4877-9E81-22AAA7EEB909, OS:11.4, name:iPhone 8 Plus }
[15:06:58]: ▸ { platform:iOS Simulator, id:E2D548B3-0527-4224-8AFA-04582D4751EC, OS:14.5, name:iPhone 8 Plus }
[15:06:58]: ▸ { platform:iOS Simulator, id:1813AB3D-3841-4CB0-9356-72BBA8784F00, OS:15.0, name:iPhone 8 Plus }
[15:06:58]: ▸ { platform:iOS Simulator, id:575FE57A-13ED-40D9-AC8E-BA9B2C413824, OS:14.5, name:iPhone 11 }
[15:06:58]: ▸ { platform:iOS Simulator, id:B9E73E1C-7E0F-4328-AA2F-908BA5FFE899, OS:15.0, name:iPhone 11 }
[15:06:58]: ▸ { platform:iOS Simulator, id:F2BF43E2-975D-4885-92EB-81E8D35973C2, OS:14.5, name:iPhone 11 Pro }
[15:06:58]: ▸ { platform:iOS Simulator, id:CC787DE2-B3F3-442E-AF7D-BAA53CF103BF, OS:15.0, name:iPhone 11 Pro }
[15:06:58]: ▸ { platform:iOS Simulator, id:3687531C-26B1-4B07-B6C9-9D624F37C7B3, OS:14.5, name:iPhone 11 Pro Max }
[15:06:58]: ▸ { platform:iOS Simulator, id:8AC46380-A3DE-4BDC-93F2-978977994DD9, OS:15.0, name:iPhone 11 Pro Max }
[15:06:58]: ▸ { platform:iOS Simulator, id:AB284AC5-CD17-4E3C-B191-AD273D2A5570, OS:14.5, name:iPhone 12 }
[15:06:58]: ▸ { platform:iOS Simulator, id:D81A0B35-A035-46F2-9417-D83696891EB8, OS:15.0, name:iPhone 12 }
[15:06:58]: ▸ { platform:iOS Simulator, id:4565C008-C95F-407A-8B3F-29C43F279921, OS:14.5, name:iPhone 12 Pro }
[15:06:58]: ▸ { platform:iOS Simulator, id:FF258455-5614-45AD-A59C-FA683AAB3E2E, OS:15.0, name:iPhone 12 Pro }
[15:06:58]: ▸ { platform:iOS Simulator, id:CC0FA84F-EDDC-4681-BA2F-A862B309472C, OS:14.5, name:iPhone 12 Pro Max }
[15:06:58]: ▸ { platform:iOS Simulator, id:3983928B-0393-4EFD-8FF4-46B7B8C1A64F, OS:15.0, name:iPhone 12 Pro Max }
[15:06:58]: ▸ { platform:iOS Simulator, id:66B8232B-AC0D-40BE-BE29-85ACEB931F4F, OS:14.5, name:iPhone 12 mini }
[15:06:58]: ▸ { platform:iOS Simulator, id:796D4EA4-29EE-40AC-9985-A1FFE36685FE, OS:15.0, name:iPhone 12 mini }
[15:06:58]: ▸ { platform:iOS Simulator, id:8B745FD2-71CD-4559-92CA-E1B723C9C7A0, OS:15.0, name:iPhone 13 }
[15:06:58]: ▸ { platform:iOS Simulator, id:073500EE-3601-401D-87F5-E738DA4522F4, OS:15.0, name:iPhone 13 Pro }
[15:06:58]: ▸ { platform:iOS Simulator, id:E3F46787-CB13-4714-B58D-01D6C0250783, OS:15.0, name:iPhone 13 Pro Max }
[15:06:58]: ▸ { platform:iOS Simulator, id:7E23B288-1A52-4F81-AE55-70A084ABE1AA, OS:15.0, name:iPhone 13 mini }
[15:06:58]: ▸ { platform:iOS Simulator, id:1CEDB664-6CAB-49DA-9DF1-92209ADE65D7, OS:11.4, name:iPhone SE (1st generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:01B3439F-D30D-4B85-BCD6-370597C6EB8C, OS:14.5, name:iPhone SE (2nd generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:8C333C0E-A13B-42CA-A36B-0743C7FCE06F, OS:15.0, name:iPhone SE (2nd generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:126ECD1B-6F40-486E-A063-AA88B6A26882, OS:11.4, name:iPhone X }
[15:06:58]: ▸ { platform:iOS Simulator, id:CCCF1141-AFBC-4D2D-B55C-44A9D1899DF4, OS:14.5, name:iPod touch (7th generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:87F10AF0-B792-4A57-A025-BA1C6C5210D1, OS:15.0, name:iPod touch (7th generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:AD993012-FA91-45AE-BD02-69C07F745DB8, OS:11.4, name:iPad (5th generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:5BD91FE9-FE43-4519-90A2-25DF829F62CE, OS:11.4, name:iPad (6th generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:F440E67C-535F-43CF-81C3-40F90E50A374, OS:14.5, name:iPad (8th generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:40C31161-780F-430D-932F-96550A292B76, OS:15.0, name:iPad (9th generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:77235528-C180-43A2-9831-766FBC72BF1E, OS:11.4, name:iPad Air }
[15:06:58]: ▸ { platform:iOS Simulator, id:34391D17-959B-4B90-B60C-8C90560C5F4E, OS:14.5, name:iPad Air (4th generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:AA88873F-1E04-4C26-9C60-036BD5B437A7, OS:15.0, name:iPad Air (4th generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:DED2C5DA-65DB-45D4-B275-526D7ED5BEBB, OS:11.4, name:iPad Air 2 }
[15:06:58]: ▸ { platform:iOS Simulator, id:3C432AA7-FF36-444E-BBC3-53C1D022B859, OS:11.4, name:iPad Pro (9.7-inch) }
[15:06:58]: ▸ { platform:iOS Simulator, id:D709EE40-9B39-42E5-B053-AABD52811715, OS:14.5, name:iPad Pro (9.7-inch) }
[15:06:58]: ▸ { platform:iOS Simulator, id:FCAEBD39-6DEA-4879-A54B-89522D6E7A91, OS:15.0, name:iPad Pro (9.7-inch) }
[15:06:58]: ▸ { platform:iOS Simulator, id:3A11DAFC-C704-4C04-B044-02FD58F42526, OS:11.4, name:iPad Pro (10.5-inch) }
[15:06:58]: ▸ { platform:iOS Simulator, id:9B99A51C-EB17-4CCE-BFC0-36A8BE9CF2B4, OS:14.5, name:iPad Pro (11-inch) (3rd generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:5DE7A2FD-7B47-467E-961C-D70DE81B23CE, OS:15.0, name:iPad Pro (11-inch) (3rd generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:108FF38F-B03A-46CD-A1EF-64AD586CC1C2, OS:11.4, name:iPad Pro (12.9-inch) (1st generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:47AE1B2D-E97B-4A19-9738-80C38AEF2690, OS:11.4, name:iPad Pro (12.9-inch) (2nd generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:54EF0873-C282-4302-92A1-38BAA1BD5D7B, OS:14.5, name:iPad Pro (12.9-inch) (5th generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:1E7D2010-1B66-4351-A199-4627810FD652, OS:15.0, name:iPad Pro (12.9-inch) (5th generation) }
[15:06:58]: ▸ { platform:iOS Simulator, id:98B8FF2E-B0FD-4D26-A3C7-E90FDAA79FDA, OS:15.0, name:iPad mini (6th generation) }
[15:06:58]: ▸ resolved source packages: 
[15:06:58]: $ xcodebuild -showBuildSettings -scheme xxx -project xxx.xcodeproj
--- xcodebuild: WARNING: Using the first of multiple matching destinations:
{ platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Any iOS Device }
{ platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Any iOS Simulator Device }
{ platform:iOS Simulator, id:73ABE1B8-2851-4508-BC24-4EE139E3E529, OS:11.4, name:iPhone 5s }
{ platform:iOS Simulator, id:ADDA00E3-C02F-45AE-BAC7-804D4DED94A3, OS:11.4, name:iPhone 6 }
{ platform:iOS Simulator, id:7E4B4D2A-2291-44E2-B6E1-5BC744074961, OS:11.4, name:iPhone 6 Plus }
{ platform:iOS Simulator, id:98A75BE0-1E9F-4BFC-8553-5517FFA2D3A3, OS:11.4, name:iPhone 6s }
{ platform:iOS Simulator, id:4C5B3EE7-F6B5-44E7-A7E1-960B470FB72D, OS:11.4, name:iPhone 6s Plus }
{ platform:iOS Simulator, id:61763B25-035F-4008-B51A-1CE1D9EF54D0, OS:11.4, name:iPhone 7 }
{ platform:iOS Simulator, id:CA248D3F-0B38-4856-AE29-E01127705DF3, OS:11.4, name:iPhone 7 Plus }
{ platform:iOS Simulator, id:5855F737-BD6A-4ECF-9609-3AB8EC2E1EA1, OS:11.4, name:iPhone 8 }
{ platform:iOS Simulator, id:368AEC8B-1298-44E7-A16A-8AA6D43AE138, OS:14.5, name:iPhone 8 }
{ platform:iOS Simulator, id:C58C1F33-63D1-490E-AE7C-5FD63DB2A40F, OS:15.0, name:iPhone 8 }
{ platform:iOS Simulator, id:4B578975-16C5-4877-9E81-22AAA7EEB909, OS:11.4, name:iPhone 8 Plus }
{ platform:iOS Simulator, id:E2D548B3-0527-4224-8AFA-04582D4751EC, OS:14.5, name:iPhone 8 Plus }
{ platform:iOS Simulator, id:1813AB3D-3841-4CB0-9356-72BBA8784F00, OS:15.0, name:iPhone 8 Plus }
{ platform:iOS Simulator, id:575FE57A-13ED-40D9-AC8E-BA9B2C413824, OS:14.5, name:iPhone 11 }
{ platform:iOS Simulator, id:B9E73E1C-7E0F-4328-AA2F-908BA5FFE899, OS:15.0, name:iPhone 11 }
{ platform:iOS Simulator, id:F2BF43E2-975D-4885-92EB-81E8D35973C2, OS:14.5, name:iPhone 11 Pro }
{ platform:iOS Simulator, id:CC787DE2-B3F3-442E-AF7D-BAA53CF103BF, OS:15.0, name:iPhone 11 Pro }
{ platform:iOS Simulator, id:3687531C-26B1-4B07-B6C9-9D624F37C7B3, OS:14.5, name:iPhone 11 Pro Max }
{ platform:iOS Simulator, id:8AC46380-A3DE-4BDC-93F2-978977994DD9, OS:15.0, name:iPhone 11 Pro Max }
{ platform:iOS Simulator, id:AB284AC5-CD17-4E3C-B191-AD273D2A5570, OS:14.5, name:iPhone 12 }
{ platform:iOS Simulator, id:D81A0B35-A035-46F2-9417-D83696891EB8, OS:15.0, name:iPhone 12 }
{ platform:iOS Simulator, id:4565C008-C95F-407A-8B3F-29C43F279921, OS:14.5, name:iPhone 12 Pro }
{ platform:iOS Simulator, id:FF258455-5614-45AD-A59C-FA683AAB3E2E, OS:15.0, name:iPhone 12 Pro }
{ platform:iOS Simulator, id:CC0FA84F-EDDC-4681-BA2F-A862B309472C, OS:14.5, name:iPhone 12 Pro Max }
{ platform:iOS Simulator, id:3983928B-0393-4EFD-8FF4-46B7B8C1A64F, OS:15.0, name:iPhone 12 Pro Max }
{ platform:iOS Simulator, id:66B8232B-AC0D-40BE-BE29-85ACEB931F4F, OS:14.5, name:iPhone 12 mini }
{ platform:iOS Simulator, id:796D4EA4-29EE-40AC-9985-A1FFE36685FE, OS:15.0, name:iPhone 12 mini }
{ platform:iOS Simulator, id:8B745FD2-71CD-4559-92CA-E1B723C9C7A0, OS:15.0, name:iPhone 13 }
{ platform:iOS Simulator, id:073500EE-3601-401D-87F5-E738DA4522F4, OS:15.0, name:iPhone 13 Pro }
{ platform:iOS Simulator, id:E3F46787-CB13-4714-B58D-01D6C0250783, OS:15.0, name:iPhone 13 Pro Max }
{ platform:iOS Simulator, id:7E23B288-1A52-4F81-AE55-70A084ABE1AA, OS:15.0, name:iPhone 13 mini }
{ platform:iOS Simulator, id:1CEDB664-6CAB-49DA-9DF1-92209ADE65D7, OS:11.4, name:iPhone SE (1st generation) }
{ platform:iOS Simulator, id:01B3439F-D30D-4B85-BCD6-370597C6EB8C, OS:14.5, name:iPhone SE (2nd generation) }
{ platform:iOS Simulator, id:8C333C0E-A13B-42CA-A36B-0743C7FCE06F, OS:15.0, name:iPhone SE (2nd generation) }
{ platform:iOS Simulator, id:126ECD1B-6F40-486E-A063-AA88B6A26882, OS:11.4, name:iPhone X }
{ platform:iOS Simulator, id:CCCF1141-AFBC-4D2D-B55C-44A9D1899DF4, OS:14.5, name:iPod touch (7th generation) }
{ platform:iOS Simulator, id:87F10AF0-B792-4A57-A025-BA1C6C5210D1, OS:15.0, name:iPod touch (7th generation) }
{ platform:iOS Simulator, id:AD993012-FA91-45AE-BD02-69C07F745DB8, OS:11.4, name:iPad (5th generation) }
{ platform:iOS Simulator, id:5BD91FE9-FE43-4519-90A2-25DF829F62CE, OS:11.4, name:iPad (6th generation) }
{ platform:iOS Simulator, id:F440E67C-535F-43CF-81C3-40F90E50A374, OS:14.5, name:iPad (8th generation) }
{ platform:iOS Simulator, id:40C31161-780F-430D-932F-96550A292B76, OS:15.0, name:iPad (9th generation) }
{ platform:iOS Simulator, id:77235528-C180-43A2-9831-766FBC72BF1E, OS:11.4, name:iPad Air }
{ platform:iOS Simulator, id:34391D17-959B-4B90-B60C-8C90560C5F4E, OS:14.5, name:iPad Air (4th generation) }
{ platform:iOS Simulator, id:AA88873F-1E04-4C26-9C60-036BD5B437A7, OS:15.0, name:iPad Air (4th generation) }
{ platform:iOS Simulator, id:DED2C5DA-65DB-45D4-B275-526D7ED5BEBB, OS:11.4, name:iPad Air 2 }
{ platform:iOS Simulator, id:3C432AA7-FF36-444E-BBC3-53C1D022B859, OS:11.4, name:iPad Pro (9.7-inch) }
{ platform:iOS Simulator, id:D709EE40-9B39-42E5-B053-AABD52811715, OS:14.5, name:iPad Pro (9.7-inch) }
{ platform:iOS Simulator, id:FCAEBD39-6DEA-4879-A54B-89522D6E7A91, OS:15.0, name:iPad Pro (9.7-inch) }
{ platform:iOS Simulator, id:3A11DAFC-C704-4C04-B044-02FD58F42526, OS:11.4, name:iPad Pro (10.5-inch) }
{ platform:iOS Simulator, id:9B99A51C-EB17-4CCE-BFC0-36A8BE9CF2B4, OS:14.5, name:iPad Pro (11-inch) (3rd generation) }
{ platform:iOS Simulator, id:5DE7A2FD-7B47-467E-961C-D70DE81B23CE, OS:15.0, name:iPad Pro (11-inch) (3rd generation) }
{ platform:iOS Simulator, id:108FF38F-B03A-46CD-A1EF-64AD586CC1C2, OS:11.4, name:iPad Pro (12.9-inch) (1st generation) }
{ platform:iOS Simulator, id:47AE1B2D-E97B-4A19-9738-80C38AEF2690, OS:11.4, name:iPad Pro (12.9-inch) (2nd generation) }
{ platform:iOS Simulator, id:54EF0873-C282-4302-92A1-38BAA1BD5D7B, OS:14.5, name:iPad Pro (12.9-inch) (5th generation) }
{ platform:iOS Simulator, id:1E7D2010-1B66-4351-A199-4627810FD652, OS:15.0, name:iPad Pro (12.9-inch) (5th generation) }
{ platform:iOS Simulator, id:98B8FF2E-B0FD-4D26-A3C7-E90FDAA79FDA, OS:15.0, name:iPad mini (6th generation) }

+--------------------------------------+------------------------------------------------------------+
|                                      Summary for gym 2.197.0                                      |
+--------------------------------------+------------------------------------------------------------+
| project                              | xxx.xcodeproj                                           |
| scheme                               | xxx                                                     |
| clean                                | true                                                       |
| export_xcargs                        | -allowProvisioningUpdates                                  |
| output_name                          | ReleaseApp                                                 |
| output_directory                     | .                                                          |
| silent                               | false                                                      |
| skip_package_ipa                     | false                                                      |
| skip_package_pkg                     | false                                                      |
| build_path                           | /Users/user/Library/Developer/Xcode/Archives/20  |
|                                      | 21-10-29                                                   |
| result_bundle                        | false                                                      |
| buildlog_path                        | ~/Library/Logs/gym                                         |
| destination                          | generic/platform=iOS                                       |
| skip_profile_detection               | false                                                      |
| skip_package_dependencies_resolution | false                                                      |
| disable_package_automatic_updates    | false                                                      |
| use_system_scm                       | false                                                      |
| xcode_path                           | /Applications/Xcode.app                                    |
+--------------------------------------+------------------------------------------------------------+

[15:07:01]: $ set -o pipefail && xcodebuild -scheme xxx -project xxx.xcodeproj -destination 'generic/platform=iOS' -archivePath /Users/user/Library/Developer/Xcode/Archives/2021-10-29/ReleaseApp\ 2021-10-29\ 15.07.01.xcarchive clean archive | tee /Users/user/Library/Logs/gym/xxx-xxx.log | xcpretty
[15:07:03]: ▸ Clean Succeeded
[15:07:05]: ▸ ❌  error: Provisioning profile "iOS Team Provisioning Profile: xxx" doesn't include signing certificate "Apple Development: user (G378254S76)". (in target 'xxx' from project 'xxx')
[15:07:05]: ▸ ** ARCHIVE FAILED **
▸ Clean Succeeded

❌  error: Provisioning profile "iOS Team Provisioning Profile: xxx" doesn't include signing certificate "Apple Development: XXX". (in target 'XXX' from project 'XXX')

** ARCHIVE FAILED **
[15:07:05]: Exit status: 65

+-------------+-------------------------+
|           Build environment           |
+-------------+-------------------------+
| xcode_path  | /Applications/Xcode.app |
| gym_version | 2.197.0                 |
| sdk         | iPhoneOS15.0.sdk        |
+-------------+-------------------------+

[15:07:05]: ▸ Build description path: /Users/user/Library/Developer/Xcode/DerivedData/xxx-cxwqbxovmsilwiagqygvczyobkgs/Build/Intermediates.noindex/ArchiveIntermediates/xxx/IntermediateBuildFilesPath/XCBuildData/d00170a6d4259ddee2f3990d3045bf8a-desc.xcbuild
[15:07:05]: ▸ note: Build preparation complete
[15:07:05]: ▸ note: Building targets in parallel
[15:07:05]: ▸ error: Provisioning profile "iOS Team Provisioning Profile: XXX" doesn't include signing certificate "Apple Development: XXX". (in target 'XXX' from project 'XXX')
[15:07:05]: 
[15:07:05]: ⬆️  Check out the few lines of raw `xcodebuild` output above for potential hints on how to solve this error
[15:07:05]: 📋  For the complete and more detailed error log, check the full log at:
[15:07:05]: 📋  /Users/user/Library/Logs/gym/XXX.log
[15:07:05]: 
[15:07:05]: Looks like fastlane ran into a build/archive error with your project
[15:07:05]: It's hard to tell what's causing the error, so we wrote some guides on how
[15:07:05]: to troubleshoot build and signing issues: https://docs.fastlane.tools/codesigning/getting-started/
[15:07:05]: Before submitting an issue on GitHub, please follow the guide above and make
[15:07:05]: sure your project is set up correctly.
[15:07:05]: fastlane uses `xcodebuild` commands to generate your binary, you can see the
[15:07:05]: the full commands printed out in yellow in the above log.
[15:07:05]: Make sure to inspect the output above, as usually you'll find more error information there

<details>
  <pre> [REPLACE THIS WITH YOUR INFORMATION] </pre>
</details>

### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 11.6                                        |
| Ruby                        | 2.6.6                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.30.1 (Apple Git-130)          |
| Installation Source         | ~/.rvm/rubies/ruby-2.6.6/bin/fastlane       |
| Host                        | macOS 11.6 (20G165)                         |
| Ruby Lib Dir                | ~/.rvm/rubies/ruby-2.6.6/lib                |
| OpenSSL Version             | OpenSSL 1.1.1j  16 Feb 2021                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 13.1                                        |
| Swift Version               | 5.5.1                                       |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#
# For a list of all available plugins, check out
#
#     https://docs.fastlane.tools/plugins/available-plugins
#

# Uncomment the line if you want fastlane to automatically update itself
# update_fastlane

default_platform(:ios)

platform :ios do
  desc "Creating your certificates for sign and upload..."
  lane :_certificates do
    get_certificates(
      development: true,
      username: "username@gmail.com"
    )
  end

  lane :_provision_profile do
    get_provisioning_profile(
      force: true,
      filename: "myFile.mobileprovision"
    )
  end

  desc "Uploading our app..."
  lane :_upload do
    gym(
      project: "xxx.xcodeproj",
      scheme: "xxx",
      clean: true,
      export_xcargs: "-allowProvisioningUpdates",
      output_name: "ReleaseApp.ipa"
    )

    # deliver(
    #   ipa: "ReleaseApp.ipa",
    #   skip_metadata: true,
    #   skip_screenshots: true,
    #   skip_app_version_update: true,
    #   force: true
    # )
  end
end
```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
# app_identifier("[[APP_IDENTIFIER]]") # The bundle identifier of your app
# apple_id("[[APPLE_ID]]") # Your Apple email address


# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.197.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                             | Version      |
| ------------------------------- | ------------ |
| rouge                           | 2.0.7        |
| xcpretty                        | 0.3.0        |
| terminal-notifier               | 2.0.0        |
| unicode-display_width           | 1.8.0        |
| terminal-table                  | 1.8.0        |
| plist                           | 3.6.0        |
| public_suffix                   | 4.0.6        |
| addressable                     | 2.8.0        |
| multipart-post                  | 2.0.0        |
| word_wrap                       | 1.0.0        |
| optparse                        | 0.1.1        |
| tty-screen                      | 0.8.1        |
| tty-cursor                      | 0.7.1        |
| tty-spinner                     | 0.9.3        |
| artifactory                     | 3.0.15       |
| babosa                          | 1.0.4        |
| colored                         | 1.2          |
| highline                        | 2.0.3        |
| commander                       | 4.6.0        |
| excon                           | 0.87.0       |
| faraday-em_http                 | 1.0.0        |
| faraday-em_synchrony            | 1.0.0        |
| faraday-excon                   | 1.1.0        |
| faraday-httpclient              | 1.0.1        |
| faraday-net_http                | 1.0.1        |
| faraday-net_http_persistent     | 1.2.0        |
| faraday-patron                  | 1.0.0        |
| faraday-rack                    | 1.0.0        |
| ruby2_keywords                  | 0.0.5        |
| faraday                         | 1.8.0        |
| unf_ext                         | 0.0.8        |
| unf                             | 0.1.4        |
| domain_name                     | 0.5.20190701 |
| http-cookie                     | 1.0.4        |
| faraday-cookie_jar              | 0.0.7        |
| faraday_middleware              | 1.2.0        |
| fastimage                       | 2.2.5        |
| gh_inspector                    | 1.1.3        |
| mini_magick                     | 4.11.0       |
| naturally                       | 2.2.1        |
| rubyzip                         | 2.3.2        |
| security                        | 0.1.3        |
| xcpretty-travis-formatter       | 1.0.1        |
| dotenv                          | 2.7.6        |
| simctl                          | 1.6.8        |
| jwt                             | 2.3.0        |
| uber                            | 0.1.0        |
| declarative                     | 0.0.20       |
| trailblazer-option              | 0.1.1        |
| representable                   | 3.1.1        |
| retriable                       | 3.1.2        |
| mini_mime                       | 1.1.2        |
| httpclient                      | 2.8.3        |
| google-apis-core                | 0.4.1        |
| google-apis-playcustomapp_v1    | 0.5.0        |
| google-apis-androidpublisher_v3 | 0.12.0       |
| google-cloud-env                | 1.5.0        |
| google-cloud-errors             | 1.2.0        |
| google-cloud-core               | 1.6.0        |
| google-apis-iamcredentials_v1   | 0.7.0        |
| google-apis-storage_v1          | 0.8.0        |
| rake                            | 13.0.6       |
| digest-crc                      | 0.6.4        |
| google-cloud-storage            | 1.34.1       |
| emoji_regex                     | 3.2.3        |
| jmespath                        | 1.4.0        |
| aws-eventstream                 | 1.2.0        |
| aws-sigv4                       | 1.4.0        |
| aws-sdk-core                    | 3.121.3      |
| aws-sdk-kms                     | 1.50.0       |
| aws-sdk-s3                      | 1.104.0      |
| rexml                           | 3.2.5        |
| nanaimo                         | 0.3.0        |
| colored2                        | 3.1.2        |
| claide                          | 1.0.3        |
| CFPropertyList                  | 3.0.4        |
| atomos                          | 0.1.3        |
| xcodeproj                       | 1.21.0       |
| json                            | 2.6.1        |
| bundler                         | 2.2.29       |
| multi_json                      | 1.15.0       |
| signet                          | 0.16.0       |
| os                              | 1.1.1        |
| memoist                         | 0.16.2       |
| googleauth                      | 1.1.0        |
| webrick                         | 1.7.0        |
| aws-partitions                  | 1.519.0      |
| forwardable                     | 1.2.0        |
| logger                          | 1.3.0        |
| date                            | 2.0.0        |
| stringio                        | 0.0.2        |
| ipaddr                          | 1.2.2        |
| openssl                         | 2.1.2        |
| zlib                            | 1.0.0        |
| mutex_m                         | 0.1.0        |
| ostruct                         | 0.1.0        |
| strscan                         | 1.0.0        |
| io-console                      | 0.4.7        |
| fileutils                       | 1.1.0        |
| etc                             | 1.0.1        |
| psych                           | 3.1.0        |
</details>

*generated on:* **2021-10-29**
</details>

<details>
  <pre> 

 </pre>
</details>

**Comment by PiotrBogus** [comment]
I got the same problem since yeasterday. I didn't change nothing on my CI.
After nuke and recreate provisioning profiles match isn't able to include certificate inside profile. On Appstore im able to see that profile contains this cert and is checked.

**Comment by KananAbilzada** [comment]
> I got the same problem since yeasterday. I didn't change nothing on my CI. After nuke and recreate provisioning profiles match isn't able to include certificate inside profile. On Appstore im able to see that profile contains this cert and is checked.

I fixed this problem with using match. (https://docs.fastlane.tools/actions/match/)
